automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookGCEDataSyncingSpec.scala

```diff
@@ -21,241 +21,244 @@ import scala.concurrent.duration._
  * and welder localization/delocalization.
  */
 @DoNotDiscover
-class NotebookGCEDataSyncingSpec extends RuntimeFixtureSpec with NotebookTestUtils {
+class NotebookGCEDataSyncingSpec extends RuntimeFixtureSpec with NotebookTestUtils with ParallelTestExecution {
   override def enableWelder: Boolean = true
 
+  override def runtimeSystemKey: Option[String] = Some(getClass.getSimpleName)
   implicit def ronToken: AuthToken = ronAuthToken.unsafeRunSync()
+  override def withFixture(test: NoArgTest) =
+    if (isRetryable(test))
+      withRetry(super.withFixture(test))
+    else
+      super.withFixture(test)
+
+  "Welder should be up" in { runtimeFixture =>
+    val resp = Welder.getWelderStatus(runtimeFixture.runtime)
+    resp.attempt.unsafeRunSync().isRight shouldBe true
+  }
 
-  "NotebookGCEDataSyncingSpec" - {
```

Comment:
all I really did here was remove one layer of indentation... not sure why git is unhappy

Reply:
I turned off whitespace comparison ... and the changes became apparent. I will comment on them below. were these ignored tests intentional?